build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
         
-        spineVersion = '0.8.31-SNAPSHOT'
+        spineVersion = '0.8.32-SNAPSHOT'
```

Comment:
Before merge, please take a look at the most recent Spine version and increase this one accordingly..